PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/coreTools/FunctionsCoreToolsManager.kt

```diff
@@ -88,7 +88,8 @@ object FunctionsCoreToolsManager {
                 } else if (SystemInfo.isWindows) {
                     AzureCoreToolsFeedReleaseFilter("Windows", listOf("x86"), listOf("minified", "full"))
                 } else if (SystemInfo.isMac && CpuArch.isArm64()) {
-                    AzureCoreToolsFeedReleaseFilter("MacOS", listOf("arm64", "x64"), listOf("full"))
+                    // TODO Maarten - Test on M1 machine AzureCoreToolsFeedReleaseFilter("MacOS", listOf("arm64", "x64"), listOf("full"))
+                    AzureCoreToolsFeedReleaseFilter("MacOS", listOf("x64"), listOf("full"))
```

Comment:
This logic is intersected now with a block below `if (SystemInfo.isMac)`.

Reply:
True. Want me to bring the two into one and just keep the TODO for myself?